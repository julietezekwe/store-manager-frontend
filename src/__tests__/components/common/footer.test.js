/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable no-undef */
import Footer from '../../../components/common/Footer.jsx';

let wrapper;
beforeEach(() => {
  wrapper = shallow(<Footer />);
});
describe('App Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
