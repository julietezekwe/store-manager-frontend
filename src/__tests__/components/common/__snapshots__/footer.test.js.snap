// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should match snapshot 1`] = `
<footer
  className="col-12"
  id="footer"
>
  <p>
    Copyright Â© 2018 Store managers Design. All rights reserved.
  </p>
</footer>
`;
